### PR TITLE
Add `restart` command for `neo-cli`

### PR DESCRIPTION
**Pull request by Liaojinghui**

### CODE REVIEW

**Comment by erikzhang** [comment]
We usually use the `screen` command to make `neo-cli` run in the background. If restarted, the `screen` process will be killed.

**Comment by Liaojinghui** [comment]
> We usually use the `screen` command to make `neo-cli` run in the background. If restarted, the `screen` process will be killed.

Will figure it out

**Comment by Liaojinghui** [comment]
https://github.com/neo-project/neo-modules/pull/626

**Comment by Liaojinghui** [comment]
image

File: neo-cli/CLI/MainService.Tools.cs [hunk 1]
```diff
@@ -52,6 +54,31 @@ private void OnParseCommand(string value)
...
+            {
+                foreach (var plugin in Plugin.Plugins) plugin.Dispose();
+
+                var psi = new ProcessStartInfo(@"dotnet")
+                {
+                    Arguments = @"neo-cli.dll",
+                    UseShellExecute = false,
+                };
+                Process.Start(psi);
+                Process.GetCurrentProcess().Kill();
```
Comment (erikzhang) [inline neo-cli/CLI/MainService.Tools.cs:73]: We can't exit normally?

**Pull request closed without merging**